src/main/python/pybuilder_aws_plugin/__init__.py

```diff
@@ -4,22 +4,35 @@
 
 import sys
 
-from pybuilder.core import init
+from pybuilder.core import init, task, depends
 
-from .lambda_tasks import upload_zip_to_s3, package_lambda_code
+from .lambda_tasks import upload_zip_to_s3, package_lambda_code, lambda_release
 
 if sys.version_info[0:2] >= (2, 7):
-    from .cfn_tasks import upload_cfn_to_s3
+    # cfn-sphere needs Python 2.7, skip cfn handling for older Python versions
+    from .cfn_tasks import upload_cfn_to_s3, cfn_release
 
+    @task(description='Release on S3 by copying everything to latest')
+    @depends('lambda_release',
+             'cfn_release')
+    def release_custom_resource():
```

Comment:
I wouldn't necessarily call this 'release_custom_resource' but rather 'aws_release'? The same goes for the 'upload_custom_resource' below. It may not always be a custom resource you are dealing with.